src/templates/controlbox.html

```diff
@@ -2,6 +2,7 @@
     <div class="chat-head controlbox-head">
         {[ if (!o.sticky_controlbox) { ]}
             <a class="chatbox-btn close-chatbox-button fa fa-times"></a>
+            <a class="chatbox-btn popout-chatbox-button fa fa-arrows-alt"></a>
```

Comment:
This button shouldn't be inside the condition, which is only to check whether a close button is relevant, if `sticky_controlbox` is `true`, then the controlbox can't be closed and the `close` button doesn't render.

We'll probably want to add a new configuration setting to determine whether the chat can pop out as well.